packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -1954,6 +1954,31 @@ describe('cluster', () => {
     });
   });
 
+  test('create a cluster from a snapshot with encrypted storage', () => {
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseClusterFromSnapshot(stack, 'Database', {
+      engine: DatabaseClusterEngine.aurora({ version: AuroraEngineVersion.VER_1_22_2 }),
+      instanceProps: {
+        vpc,
+      },
+      snapshotIdentifier: 'mySnapshot',
+      storageEncryptionKey: new kms.Key(stack, 'Key'),
```

Comment:
How about using an imported Key in the test (`Key.fromKeyArn()`)? I think that will make the test read nicer.

Reply:
Done